readme.md

```diff
@@ -372,6 +372,12 @@ As in: OOPs I used Java!
 
 ---
 
+**Q:** Why aren't crypto engineers allowed inside bars?
+
+**A:** They're miners!
```

Comment:
I don't get it?

Reply:
play on the word "miner" (crypto) vs "minor" (non-adult).

Reply:
I understand now. I think there's some room for improvement since crypo engineer is too broad. Perhaps you are talking about cryptocurrency.

Also, maybe theres a better way to use minor instead of bar. Like non-adult or child.

Reply:
@styfle ok, I reworked it. Still cheesy but non-alcoholic (virgin) version now. More explicit about cryptocurrency vs. crypto.